### DISCUSSION TITLE
Does the FABGL library support UTF-8 on FABGL board ?

### DISCUSSION THREAD

**Discussion by andromeda2911** [fdivitto/FabGL#317] [Q&A]
Hi,
I need to print text file utf-8 on terminal.
Does the FABGL library support UTF-8 ?
Thanks.

**Comment by andromeda2911** [comment]
I think is not possible because it's impossible to change encoding, for example 
`fr_FR.ISO-8859`

**Comment by JacekRuzyczka** [comment]
@andromeda2911 Where do you set character encoding? The string you stated reminds me of the `LC_CTYPE` environment variable on Unix systems, so you make the _host_ encode the data it sends in ISO Latin 1. Please note that some Unix installations ignore such statements and keep on talking in UTF-8.

Whether the text file in question is also encoded in Latin 1, is another question.

**Comment by andromeda2911** [comment]
I'm using arduino on windows, and the terminal is fabgl, not unix.
